swap/protocol.go

```diff
@@ -147,6 +146,12 @@ func (s *Swap) getPeer(id enode.ID) (*Peer, bool) {
 	return peer, ok
```

Comment:
as in #1701, just return `peer` and do away with the `ok` variable (we can check for `nil`).

Reply:
done